README.md

```diff
@@ -1,20 +1,27 @@
-# Project Name
+# 01-FCC-CAT-PHOTO-APP
 
-> One paragraph statement about the project.
+> A CAT PHOTO APP BUILT USING HTML.
```

Comment:
There is no need for space between your project name and statement about the project